beastify/popup/choose_beast.css

```diff
@@ -6,25 +6,28 @@ html, body {
   display: none;
 }
 
-button {
+.button {
   border: none;
   width: 100%;
   margin: 3% auto;
   padding: 4px;
   text-align: center;
   font-size: 1.5em;
   cursor: pointer;
-  background-color: #E5F2F2;
 }
 
-button:hover {
+.beast:hover {
   background-color: #CFF2F2;
 }
 
-button[type="reset"] {
+.beast {
```

Comment:
is there a reason we to add classes? Is there something about extensions that make type selectors and attribute selectors not work??

Reply:
Or is this just reverting 502?